.travis.yml

```diff
@@ -32,6 +33,7 @@ script:
   - if [[ $coverage = 1 ]]; then phpdbg -qrr vendor/bin/behat --profile coverage; else vendor/bin/behat; fi
   - if [[ $coverage = 1 ]]; then phpdbg -qrr phpcov.phar merge --clover build/logs/clover.xml build/cov; fi
   - tests/Fixtures/app/console api:swagger:export > swagger.json && swagger validate swagger.json && rm swagger.json
+  - if [[ $lint = 1 ]]; then php php-cs-fixer.phar fix --dry-run --diff; fi
```

Comment:
Maybe we could lint before tests? Also, if lint fails, we could skip tests.

If it's not the point, why adding the linter to travis if styleci is already taking care of this?